src/app/components/cards/Comment.scss

```diff
@@ -184,17 +189,35 @@
 
 @media screen and (max-width: 39.9375em) {
   .root {
-    .Comment__header, .Comment__footer, .Comment__body, .Comment__replies {
+    .Comment__block .Comment__header,
+    .Comment__block .Comment__footer,
+    .Comment__block .Comment__body,
+    .Comment__replies {
       margin-left: 0;
```

Comment:
This looks to be just a dupe now

Reply:
@roadscape I wasn't sure if the margin-left needed to also be outside of the  @include themify($themes) block or not in case the theme classes are not added

Reply:
@roadscape I removed them